config/owasp/suppressions.xml

```diff
@@ -1,5 +1,32 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
+  <suppress>
```

Comment:
Do we need them?

Reply:
Removing them as they are no longer required